package-lock.json

```diff
@@ -31656,6 +31676,12 @@
         "xml-js": "^1.6.11"
       }
     },
+    "node_modules/myst-to-jats/node_modules/nbtx": {
+      "version": "0.4.0",
+      "resolved": "https://registry.npmjs.org/nbtx/-/nbtx-0.4.0.tgz",
+      "integrity": "sha512-nSlBJ3pcSGS6qryzi1BiutEqGwc39IJMWUOjKFX3jLOW3BZS4tq0/G1+uEtQohQEKya4uago7/gLASbSMNmZ8w==",
+      "license": "MIT"
+    },
```

Comment:
This is something to look out for in the future. We are using two versions of `nbtx` @fwkoch in `myst-to-jats`.